htdocs/projet/tasks/contact.php

```diff
@@ -270,204 +276,13 @@
 
 		dol_fiche_end();
 
-		/*
-		 * Lignes de contacts
-		 */
-		print '<br>';
-/*
 		// Contacts lines (modules that overwrite templates must declare this into descriptor)
 		$dirtpls=array_merge($conf->modules_parts['tpl'],array('/core/tpl'));
 		foreach($dirtpls as $reldir)
 		{
-		    $res=@include dol_buildpath($reldir.'/contacts.tpl.php');
-		    if ($res) break;
+			$res=@include dol_buildpath($reldir.'/contacts.tpl.php');
```

Comment:
We are supposed to use tabs instead of spaces... PHPStorm's fault probably. Should I change back to spaces?